azkaban-common/src/main/java/azkaban/metrics/CommonMetrics.java

```diff
@@ -34,6 +34,8 @@
   private final MetricsManager metricsManager;
   private Meter dbConnectionMeter;
   private Meter flowFailMeter;
+  private Meter dispatchFailMeter;
```

Comment:
I think we should have a better way of organizing metrics. Keeping all metrics together in one class seems like putting apples and oranges in one basket. Also, such organization of metrics hints that metrics events can be declared from anywhere else in the code. However, dispatch fail / success should only be controlled by the `ExecutorManager`. Thoughts?